litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowGraphs/index.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Paper,
+  Select,
+} from '@material-ui/core';
+import { RadialChart } from 'litmus-ui';
+import React, { useState } from 'react';
+import { Filter } from '../../../../models/graphql/scheduleData';
+import {
+  ExecutionData,
+  ScheduledWorkflows,
+} from '../../../../models/graphql/workflowListData';
+import ScheduleAndRunStats from './ScheduleAndRunStats';
+import useStyles from './styles';
+
+interface WorkflowGraphsProps {
+  data: ScheduledWorkflows | undefined;
+}
+
+interface RadialChartMetric {
+  value: number;
+  label?: string;
+  baseColor?: string;
+}
+
+const WorkflowGraphs: React.FC<WorkflowGraphsProps> = ({ data }) => {
+  const classes = useStyles();
+
+  let completed = 0;
+  let running = 0;
+  let failed = 0;
+
+  /* eslint-disable no-unused-expressions */
+  data?.ListWorkflow.workflows.map((datarow) =>
+    datarow.workflow_runs?.map((runs) => {
+      if (
+        (JSON.parse(runs.execution_data) as ExecutionData).phase === 'Succeeded'
+      )
+        completed++;
+      else if (
+        (JSON.parse(runs.execution_data) as ExecutionData).phase === 'Running'
+      )
+        running++;
+      else if (
+        (JSON.parse(runs.execution_data) as ExecutionData).phase === 'Failed'
+      )
+        failed++;
+    })
+  );
+
+  // States for filters
+  const [filters, setFilters] = useState<Filter>(Filter.monthly);
+
+  const graphData: RadialChartMetric[] = [
+    { value: completed, label: 'Completed', baseColor: '#00CC9A' },
+    { value: running, label: 'Running', baseColor: '#5252F6' },
+    { value: failed, label: 'Failed', baseColor: '#CA2C2C' },
```

Comment:
can we pick from theme

Reply:
I checked, it's not available in the theme

Reply:
for now we can use this but we need to add to theme cc: @arkajyotiMukherjee @S-ayanide @whitetiger1399